frontend/packages/kubevirt-plugin/integration-tests/tests/utils/consts.ts

```diff
@@ -64,18 +64,20 @@ export const INNER_TEMPLATE_VERSION = 'v0.9.1';
 export const COMMON_TEMPLATES_NAMESPACE = 'openshift';
 export const COMMON_TEMPLATES_REVISION = '1';
 
+export const DEFAULT_YAML_VM_NAME = 'vm-example';
+
 export const KUBEVIRT_SCRIPTS_PATH =
   './packages/kubevirt-plugin/integration-tests/tests/utils/scripts';
 export const KUBEVIRT_TEMPLATES_PATH =
   './packages/kubevirt-plugin/integration-tests/tests/utils/templates';
 
 export enum TAB {
-  Consoles = 'Consoles',
+  Overview = 'Overview',
```

Comment:
Is the ordering here important or are you just trying to keep it the same as the ordering in the window? If it's important, I'd add a comment above it to indicate that so that it doesn't get changed inadvertently in the future.